pilot/pkg/model/xds_cache.go

```diff
@@ -15,13 +15,51 @@
 package model
 
 import (
+	"fmt"
+	"istio.io/istio/pilot/pkg/features"
+	"os"
 	"sync"
 
 	"github.com/golang/protobuf/ptypes/any"
 
+	"github.com/hashicorp/golang-lru"
+
 	"istio.io/istio/pilot/pkg/util/sets"
+	"istio.io/pkg/monitoring"
+)
+
+var (
+	reads = monitoring.NewSum(
+		"xds_cache_reads",
+		"Total number of xds cache reads.",
+		monitoring.WithLabels(typeTag),
+	)
+
+	hits   = reads.With(typeTag.Value("hit"))
```

Comment:
Does this work? I think you need to register it with the stats system with `monitoring.MustRegister` but not 100% sure

Reply:
👍  thanks